strongbox-data-service/src/main/java/org/carlspring/strongbox/data/server/EmbeddedOrientDbServer.java

```diff
@@ -141,6 +261,12 @@ private String getDatabasePath()
         return getVaultDirectory() + "/db";
     }
 
+    private String getStudioPath()
+    {
+        return getVaultDirectory() + "/www";
```

Comment:
Could we call this something more like... `orientdb-studio`...?

Reply:
It called so in OrietntDB so I think we should follow this conversion.